middleware/cache/cache_test.go

```diff
@@ -231,6 +231,53 @@ func Test_Cache_NothingToCache(t *testing.T) {
 	}
 }
 
+func Test_Cache_CustomNext(t *testing.T) {
+	app := fiber.New()
+
+	app.Use(New(Config{
+		Next: func(c *fiber.Ctx) bool {
+			if c.Response().StatusCode() == fiber.StatusOK {
+				return  false
+			}
+			return true
+		},
+	}))
+
+	app.Get("/", func(c *fiber.Ctx) error {
+		return c.SendString(time.Now().String())
+	})
+
+	app.Get("/error", func(c *fiber.Ctx) error {
+		return c.Status(fiber.StatusInternalServerError).Send([]byte(time.Now().String()))
```

Comment:
Why do you use Send here but not SendString?

Reply:
Oh,  I will change that.